dir.props

```diff
@@ -66,17 +66,18 @@
   <Import Project="$(MSBuildThisFileDirectory)dependencies.props" />
 
   <!-- list of nuget package sources passed to nuget restore -->
-  <ItemGroup Condition="'$(ExcludeInternetFeeds)' != 'true'">
-    <!-- Example to consume local CoreCLR package:
-         /p:OverridePackageSource=C:\coreclr\bin\Product\Windows_NT.x64.Debug\.nuget\pkg
-    -->
-    <DotnetSourceList Include="$(OverridePackageSource)" />
-    <!-- Need to escape double forward slash (%2F) or MSBuild will normalize to one slash on Unix. -->
-    <!-- Including buildtools to pull in TestSuite packages and repackaged xunit dependencies-->
-    <DotnetSourceList Include="https:%2F%2Fdotnet.myget.org/F/dotnet-buildtools/api/v3/index.json" />
-    <DotnetSourceList Include="https:%2F%2Fdotnet.myget.org/F/dotnet-core/api/v3/index.json" />
-    <DotnetSourceList Include="https:%2F%2Fapi.nuget.org/v3/index.json" />
-  </ItemGroup>
+  <!-- Example to consume local CoreCLR package:
+       /p:OverridePackageSource=C:\coreclr\bin\Product\Windows_NT.x64.Debug\.nuget\pkg
+       -->
+  <PropertyGroup>
+    <RestoreSources Condition="'$(DotNetBuildOffline)' != 'true'">
+      https://dotnet.myget.org/F/dotnet-buildtools/api/v3/index.json;
+      https://dotnet.myget.org/F/dotnet-core/api/v3/index.json;
+      https://api.nuget.org/v3/index.json;
+      $(OverridePackageSource);
```

Comment:
@weshaggard Is this working well with `/p:OverridePackageSource=https://tizen.myget.org/F/dotnet-core/api/v3/index.json`  option from command line ?

Reply:
I believe this isn't working because BuildTools master doesn't have this file from BuildTools release/2.0.0, which maps the `RestoreSources` to `DotnetSourceList` items: https://github.com/dotnet/buildtools/blob/release/2.0.0/src/Microsoft.DotNet.Build.Tasks/PackageFiles/RepoAPI.Mapping.props

Reply:
@dagood Thanks for the information!

@dagood @weshaggard Do you have any idea to resolve this issue ?

Reply:
We can resolve this issue 2 ways:

1. Merge https://github.com/dotnet/buildtools/pull/1707 into buildtools/master.
2. Add this workaround like we did in core-setup:  https://github.com/dotnet/core-setup/blob/d588ce3b5dfb9b4d919fda398d78abdd7a705150/dir.props#L114-L117

My opinion is we should take (1).

Reply:
I'll submit a BuildTools PR with the commits we need.

Reply:
Yes we need to get all the necessary changes into BuildTools master. I would have expected a lot more things to fail during restore without that change, we should try to understand why the others worked.

Reply:
I don't think there were _that_ many changes to buildtools release/2.0... I skimmed the commits and the ones @dagood is porting seems to be all that are needed.

Reply:
Here's the CoreFX PR to pick up the new BuildTools: https://github.com/dotnet/corefx/pull/24841.

Reply:
Thanks @dagood. I will merge on top of that but first I want to figure out why the restore worked on any of the platforms. We must still have some feeds getting pulled in somewhere.

Reply:
The only changes happened around restore, and the 2.0.0 RTM CLI tooling we're using now picks up this `RestoreSources` property, so the basic scenarios continued to work. The problem we hit here seems to be because we aren't passing `OverridePackageSource` to `dotnet restore`.

@eerhardt fixed this for some other properties with https://github.com/dotnet/buildtools/pull/1749... I think we're actually going to need to add `OverridePackageSource` there, too.